SConstruct

```diff
@@ -402,8 +402,8 @@ if platform == "osx":
     libfmod = 'libfmod%s.dylib' % lfix
     libfmodstudio = 'libfmodstudio%s.dylib' % lfix
     cpp_bindings_libname += '.a'
-    env.Append(CPPPATH=[env['headers_dir'], env['cpp_bindings_dir'] + 'include/', env['cpp_bindings_dir'] + 'include/core/',
-               env['cpp_bindings_dir'] + 'include/gen/', env['fmod_lib_dir'] + 'osx/core/inc/', env['fmod_lib_dir'] + 'osx/studio/inc/'])
+    env.Append(CPPPATH=[env['headers_dir'], env['cpp_bindings_dir'] + '/include/', env['cpp_bindings_dir'] + '/include/godot_cpp/', env['cpp_bindings_dir'] + '/include/godot_cpp/core/',
```

Comment:
A bit messy all of that but it makes the code compile for me at least. This file probably requires another cleanup.